cmd/cloud/user.go

```diff
@@ -41,6 +47,36 @@ func newUserInviteCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newUserListCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"inv"},
+		Short:   "List all the users in your Astro Organization",
+		Long:    "List all the users in your Astro Organization",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return listUsers(cmd, out)
+		},
+	}
+	cmd.Flags().IntVarP(&limit, "limit", "l", limitDefault, "Maximum number of organization users listed")
```

Comment:
I could be wrong, but I think the implementation of the limit is slightly misleading because as of now we would list all the users in the organization irrespective of the limit passed. This limit value as of now is only being used in the API call pagination, but we always end up fetching all the users in the end.

Reply:
so should we not expose the limit to the user? what to you think the limit should be?

Reply:
Either we can only output the number of records equal to the limit passed by the user and exit or we can use limit as page size and paginate the whole response, i.e. if the limit is 10, then we list the first 10 users, then ask for user input for next 10 sets of users or exit like a web page. The software side has something similar, reference: https://github.com/astronomer/astro-cli/pull/662
If we are not comfortable with either then we shouldn't allow users to pass the limit at all.

Reply:
I would like to just list all the users by default. I don't see any organization have too many users for that. How would we implement that? what should we set the limit too?

Reply:
well in that case we could have a limit specified inside the code only (could be 100 to start with), and don't expose it to the users via flag since it is more of an internal CLI logic rather than a UX option.